nessie/golden/lakectl_repo_create.golden

```diff
@@ -0,0 +1,5 @@
+Repository: lakefs://${REPO}
+Repository '${REPO}' created:
+storage namespace: ${STORAGE}
+default branch: ${BRANCH}
+timestamp: <TIMESTAMP>
```

Comment:
Odd: why is there no newline here?  (Is it a bug in `repo create`?)

Reply:
It does have a newline, but a single one.
Other commands have an additional newline, adding an empty line between the output and the next prompt.
I think we can open an issue to investigate that. WDYT?